aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -338,25 +356,34 @@ module aptos_framework::account {
 
         // Verify the given `from_public_key_bytes` matches this account's current authentication key.
         if (from_scheme == ED25519_SCHEME) {
-            let from_pk = ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
-            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
+            let from_pk =
+                ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
+            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(
+                &from_pk
+            );
             assert!(
                 account_resource.authentication_key == from_auth_key,
-                error::unauthenticated(EWRONG_CURRENT_PUBLIC_KEY)
+                error::unauthenticated(EWRONG_CURRENT_PUBLIC_KEY),
             );
         } else if (from_scheme == MULTI_ED25519_SCHEME) {
-            let from_pk = multi_ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
-            let from_auth_key = multi_ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
+            let from_pk =
+                multi_ed25519::new_unvalidated_public_key_from_bytes(
+                    from_public_key_bytes
+                );
+            let from_auth_key =
+                multi_ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
```

Comment:
Interesting here, maybe we should make max characters per line a little longer, so it doesn't wrap like this.

Reply:
I agree with your idea, and we could also reserve some extra space internally, for example, 8 characters of redundancy